entries/f/firstdirect.com.json

```diff
@@ -1,14 +1,16 @@
 {
   "First Direct": {
-    "domain": "www1.firstdirect.com",
-    "img": "www1.firstdirect.com.png",
+    "domain": "firstdirect.com",
```

Comment:
Looks like `https://firstdirect.com` isn't set up correctly. It will redirect to either the `www1` or `www2` subdomain but it does this using HTTP.
![](https://camo.githubusercontent.com/de0d204c28bb8c6054ccc844b253166a2067f327d87d711e98c875a5ffbd3773/68747470733a2f2f642e74656d7066696c65732e646f776e6c6f61642f44363044393136303834394430372f3f703d75453746495735673655616b)
I suggest using `www2.firstdirect.com` as a `url` tag.
```suggestion
    "domain": "firstdirect.com",
    "url": "www2.firstdirect.com",
```